neuprint/neuroncriteria.py

```diff
@@ -160,6 +162,17 @@ def __init__( self, matchvar='n', *,
             In either case, use use ``roi_req`` to specify whether a neuron must match just
             one (``any``) of the listed ROIs, or ``all`` of them.
 
+            **Matching against missing values (NULL)**
```

Comment:
Perhaps this new text should be split into a separate note, just to separate the null handling stuff from the ROI stuff?  (I think rst permits consecutive notes...)

```rst
        .. note::

            **Matching against missing values (NULL)**
```